worlds/sc2wol/docs/en_Starcraft 2 Wings of Liberty.md

```diff
@@ -2,36 +2,39 @@
 
 ## Where is the settings page?
 
-The [player settings page for this game](../player-settings) contains all the options you need to configure and export a
-config file.
+Follow [this link](../player-settings) to reach the player settings page for this game.
 
 ## What does randomization do to this game?
 
-Items which the player would normally acquire throughout the game have been moved around. Logic remains, so the game is
-always able to be completed.  Options exist to also shuffle around the mission order of the campaign.
+The following unlocks are randomized as items:
+1. Your ability to build any non-worker unit (including Marines!).
+2. Your ability to upgrade infantry weapons, infantry armor, vehicle weapons, etc.
+3. All armory upgrades
+4. All laboratory upgrades
+5. All mercenaries
+6. Small boosts to your starting mineral and vespene gas totals on each mission
 
-## What is the goal of Starcraft 2 when randomized?
+You find items by making progress in bonus objectives (like by rescuing allies in 'Zero Hour') and by completing
+missions. When you receive items, they will immediately become available, even during a mission, and you will be
+notified via a text box in the top-right corner of the game screen. (The text client for StarCraft 2 also records all
+items in all worlds.)
 
-The goal remains unchanged. Beat the final mission All In.
+You can also choose to shuffle the mission order, and which variant of 'All In' you must complete.
```

Comment:
The game info page is more suited to a general overview of the game and not necessarily all of the different options available to be randomized. The settings page has tooltips explaining all of the different settings so I would prefer this page to be a lot shorter and more concise.

Reply:
I've removed this line.

On a philosophical note, my understanding of the Game Page is that it's a sort of from-nothing introduction to any given randomizer. Nobody will go to these pages once they've installed and configured their game, because the page offers nothing new to them. So I think our target audience should be utter newbies (whom we can still expect to be familiar with the original game), and imo, it's worth mentioning specific gameplay-altering randomizer options, because that informs the user what the randomizer can be like. Obviously stuff like "you can change your sprite to Charizard too!" is completely unnecessary and shouldn't be on the Game Page, but I think it's not overexplaining to mention the option to shuffle mission order at the very least.

Reply:
On the note of making the entire page more concise, most edits I can think of would either damage the readability or eliminate information that actually isn't communicated on the settings page (because none of the enumerated items are controllable, so they don't appear on the settings page at all). Of course, please feel free to suggest specific changes. :)